.github/workflows/tests.yml

```diff
@@ -62,10 +62,10 @@ jobs:
         run: python manage.py test --shuffle
 
   mypy:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     container:
-      image: python:3.8
+      image: python:3.10
```

Comment:
Any reason why we still have the `container` here? We use `setup_python` below and I don't see why mypy would be a reason to use the container now. Maybe removing these lines will fix CI?